Linked_List/StartOfLoop.java

```diff
@@ -0,0 +1,94 @@
+import java.util.Scanner;
```

Comment:
Please add problem statement on top of the code.

Reply:
I have made the required changes.